crawler/lookingglass_crawler/spiders/media_spider.py

```
@@ -107,16 +107,17 @@ def start_requests(self):
                         'DNT': '1',
                     },
                     errback=self.handle_error,
-                    dont_filter=False
+                    dont_filter=True  # Always crawl start URLs, bypass DeltaFetch
                 )
             except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)